This patch applies to the zplug installation.
The first change solve(ignores) a security issue that sometimes crops up
with zsh completions.

The next two are performance improvements.

diff --git a/base/core/core.zsh b/base/core/core.zsh
index 4e2624f..74c8254 100644
--- a/base/core/core.zsh
+++ b/base/core/core.zsh
@@ -153,7 +153,7 @@ __zplug::core::core::prepare()

     # Run compinit if zplug comp file hasn't load
     if (( ! $+functions[_zplug] )); then
-        compinit -C -d "$ZPLUG_HOME/zcompdump"
+        compinit -u -C -d "$ZPLUG_HOME/zcompdump"
     fi
 }

diff --git a/base/job/polling.zsh b/base/job/polling.zsh
index a9cea49..42993c2 100644
--- a/base/job/polling.zsh
+++ b/base/job/polling.zsh
@@ -8,9 +8,9 @@ __zplug::job::polling::periodic()
         if [[ -o monitor ]]; then
             return 0
         fi
-        if setopt monitor; then
-            __zplug::log::write::info "turn monitor on"
-        fi
+        #if setopt monitor; then
+            #__zplug::log::write::info "turn monitor on"
+        #fi
     fi
 }

diff --git a/base/log/write.zsh b/base/log/write.zsh
index f94eb7c..282a28c 100644
--- a/base/log/write.zsh
+++ b/base/log/write.zsh
@@ -14,7 +14,7 @@ __zplug::log::write::debug()

 __zplug::log::write::info()
 {
-    __zplug::job::handle::flock --escape \
-        "$_zplug_log[trace]" \
-        "$(__zplug::log::format::with_json "INFO" "$argv[@]")"
+    # __zplug::job::handle::flock --escape \
+        # "$_zplug_log[trace]" \
+        # "$(__zplug::log::format::with_json "INFO" "$argv[@]")"
 }
